Fix misleading message from configure saying it is checking for zlib in /usr
when it isn't.
--- configure.orig	2023-12-02 20:22:30.000000000 -0600
+++ configure	2023-12-11 00:10:53.000000000 -0600
@@ -6104,7 +6104,7 @@
 # zlib test
 # ripped from http://autoconf-archive.cryp.to/check_zlib.html
 #
-ZLIB_HOME=/usr/local
+ZLIB_HOME=@PREFIX@
 if test ! -f "${ZLIB_HOME}/include/zlib.h"
 then
         ZLIB_HOME=/usr
